docs/baseplate/core.rst

```diff
@@ -58,6 +58,38 @@ integrations <integration/index>`.
 .. autoclass:: TraceInfo
    :members: from_upstream
 
+Authentication
+--------------
+
+If the application is dealing with authentication either directly from the
```

Comment:
"dealing with authentication" maybe call out why you would want to deal with authn, like "if the application wants to check the authenticated user for a given request to authorize data access..."